test/swivel/Swivel.sol

```diff
@@ -583,8 +587,12 @@ contract Swivel {
   /// @param c Compounding token address
   /// @param a Amount to withdraw
   function withdraw(uint8 p, address c, uint256 a) internal returns (bool) {
-    if (p == uint8(Protocols.Compound)) {
-      return ICompoundToken(c).redeemUnderlying(a) == 0;
+    // TODO as stated elsewhere, we may choose to simply return true here and not attempt to measure against any expected return
+    if (p == uint8(Protocols.Compound)) { // TODO is Rari a drop in here?
+      return ICompound(c).redeemUnderlying(a) == 0;
+    } else if (p == uint8(Protocols.Yearn)) {
+      // yearn vault api states that withdraw returns uint256
+      return IYearn(c).withdraw(a) >= 0;
```

Comment:
Early optimization, but an easy one. `uint256`s tested for `>=` to 0 can be more efficiently checked via the `!=` operator.
```suggestion
    } else if (p == uint8(Protocols.Yearn)) {
      // yearn vault api states that withdraw returns uint256
      return IYearn(c).withdraw(a) != 0;
```

Reply:
0 is a `uint256` allowable value and I don't know that `0` would never be returned. Thus `!= 0` may revert on an acceptable value.